src/components/MovieCard/MovieCard.jsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import propTypes from 'prop-types';
+import './MovieCard.scss';
+
+export const MovieCard = ({
+  imgUrl,
+  title,
+  description,
+  imdbUrl,
+}) => (
+  <div className="card">
+    <div className="card-image">
+      <figure className="image is-4by3">
+        <img
+          // eslint-disable-next-line max-len
+          src={imgUrl}
+          alt="Film logo"
+        />
```

Comment:
```suggestion
        <img src={imgUrl} alt="Film logo" />
```
2 атрибута пишем в одну строку, как в html